coordinator/src/main/java/org/apache/uniffle/coordinator/CoordinatorGrpcService.java

```diff
@@ -196,13 +196,14 @@ public void reportClientOperation(
   }
 
   @Override
-  public void appHeartbeat(
-      AppHeartBeatRequest request,
-      StreamObserver<AppHeartBeatResponse> responseObserver) {
+  public void registerApplicationInfo(
```

Comment:
Why do we rename `appHeartbeat` to `registerApplicationInfo`? I mean that we can add a method `registerApplicationInfo`.

Reply:
> Maybe we could add a rpc method called registerApplicationInfo for Coordinator.

Didn't it say to add a new rpc method? If not, the server still needs to send the user's attribute

Reply:
`CoordinatorCrpcService` should have `heartbeat` and `registerApplicationInfo` methods  at the same time.

Reply:
You mean to keep sendAppHeartbeat, and use registerApplicationInfo to send a request to the coordinator only once for recording. And still use sendAppHeartbeat later, right?

Reply:
Done.